client/ayon_core/plugins/publish/extract_oiio_postprocess.py

```diff
@@ -0,0 +1,338 @@
+from __future__ import annotations
+from typing import Any, Optional
+import os
+import copy
+import clique
+import pyblish.api
+
+from ayon_core.pipeline import (
+    publish,
+    get_temp_dir
+)
+from ayon_core.lib import (
+    is_oiio_supported,
+    get_oiio_tool_args,
+    run_subprocess
+)
+from ayon_core.lib.profiles_filtering import filter_profiles
+
+
+class ExtractOIIOPostProcess(publish.Extractor):
+    """Process representations through `oiiotool` with profile defined
+    settings so that e.g. color space conversions can be applied or images
+    could be converted to scanline, resized, etc. regardless of colorspace
+    data.
+    """
+
+    label = "OIIO Post Process"
+    order = pyblish.api.ExtractorOrder + 0.020
+
+    settings_category = "core"
+
+    optional = True
+
+    # Supported extensions
+    supported_exts = {"exr", "jpg", "jpeg", "png", "dpx"}
```

Comment:
any reason why not to use [IMAGE_EXTENSIONS](https://github.com/ynput/ayon-core/blob/5bccc7cf2bcdc51e446cf7794f09d6081ff7255a/client/ayon_core/lib/transcoding.py#L49) - too broad perhaps for OIIO?